samza-core/src/main/java/org/apache/samza/execution/JobPlanner.java

```diff
@@ -127,9 +134,6 @@ final void writePlanJsonFile(String planJson) {
 
   private Map<String, String> getGeneratedConfig(String runId) {
```

Comment:
does this method still need a runId passed into it?

Reply:
nice catch, cleaned